doc/2/api/controllers/auth/get-current-user/index.md

```diff
@@ -16,11 +16,20 @@ Returns information about the currently logged in user.
 
 ### HTTP
 
+<SinceBadge version="auto-version"/>
+```http
+URL: http://kuzzle:7512/_me/users
```

Comment:
```suggestion
URL: http://kuzzle:7512/_me
```

Reply:
We don't need to add `users` here IMHO

Reply:
Like it better, _'users'_ made no sense.